controllers/articles.controller.js

```diff
@@ -2,13 +2,18 @@ const { fetchArticleById, fetchArticles, fetchCommentsOnArticle, addCommentToArt
 
 exports.getArticles = (request, response, next) => {
     const {sort_by} = request.query
-    return fetchArticles(sort_by)
+    const {order} = request.query
+    return fetchArticles(sort_by, order)
     .then((articles) => {
         articles.forEach((article) => {
             article.comment_count = parseInt(article.comment_count) //<< change comment_count to number from string
```

Comment:
There's no need to change the comment count from a string to a number. If you really want to do this, you can convert it to an INT in the SQL query. It is really not needed though